demos/object_detection_demo_ssd_async/main.cpp

```diff
@@ -50,523 +135,127 @@ bool ParseAndCheckCommandLine(int argc, char *argv[]) {
-
-                request = std::move(emptyRequests.front());
-                emptyRequests.pop_front();
-            }
-            frameToBlob(frame, request, imageInputName);
-
-            ExecutionMode frameMode = currentMode;
-            request->SetCompletionCallback([&mutex,
-                                            &completedRequestResults,
-                                            nextFrameId,
-                                            frame,
-                                            request,
-                                            &outputName,
-                                            startTime,
-                                            frameMode,
-                                            &emptyRequests,
-                                            &callbackException,
-                                            &condVar] {
-                {
-                    std::lock_guard<std::mutex> callbackLock(mutex);
-
-                    try {
-                        completedRequestResults.insert(
-                            std::pair<int, RequestResult>(nextFrameId, RequestResult{
-                                frame,
-                                std::make_shared<TBlob<float>>(*as<TBlob<float>>(request->GetBlob(outputName))),
-                                startTime,
-                                frameMode
-                            }));
-
-                        emptyRequests.push_back(std::move(request));
-                    }
-                    catch (...) {
-                        if (!callbackException) {
-                            callbackException = std::current_exception();
-                        }
-                    }
-                }
-                condVar.notify_one();
-            });
-
-            request->StartAsync();
-            nextFrameId++;
         }
-        // -----------------------------------------------------------------------------------------------------
 
-        // --------------------------- 8. Report metrics -------------------------------------------------------
+        //// --------------------------- Report metrics -------------------------------------------------------
+        auto info = pipeline.getPerformanceInfo();
         slog::info << slog::endl << "Metric reports:" << slog::endl;
 
-        /** Show performance results **/
-        if (FLAGS_pc) {
-            if (currentMode == ExecutionMode::USER_SPECIFIED) {
-                for (const auto& request: userSpecifiedInferRequests) {
-                    printPerformanceCounts(*request, std::cout, getFullDeviceName(ie, FLAGS_d));
-                }
-            } else printPerformanceCounts(*minLatencyInferRequest, std::cout, getFullDeviceName(ie, FLAGS_d));
-        }
+        std::cout << std::endl << "Total time: "
+            << std::chrono::duration_cast<std::chrono::milliseconds> (std::chrono::steady_clock::now()- startTimePoint).count()
+            << " ms" << std::endl;
```

Comment:
I think we don't need this.

Reply:
that was in original demo AFAIR.

Reply:
I don't see how it is useful. And this demo will be sort of a template so every other demo will bring that useless report.

Reply:
Total time is useful to measure full processing time, including "cold start" time

Reply:
And how is it useful?
I haven't yet viewed `PerformanceInfo`, but it seems that it starts counting with the first `submitImage()` which is the first thing main loop does. This means that `Total time` is equivalent to `FPS * lastFrameId`. You may interpret FPS as Total time normalized by video length.

Reply:
That makes sence. But as long as it was in original demo, I want to get approval from all stakeholders before removing it. Please arrange discussion via mail.

Reply:
That's ridiculously big leap of escalation. I think first we need to ask @vladimir-dudnik, @eizamaliev, @IRDonch.

Reply:
Agreed. Go on.

Reply:
have no objection from removing this

Reply:
OK, removed